src/Microsoft.AspNet.Antiforgery/Resources.resx

```diff
@@ -152,8 +152,14 @@
   <data name="Antiforgery_FormToken_MustBeProvided" xml:space="preserve">
     <value>The required antiforgery form field "{0}" is not present.</value>
   </data>
-  <data name="Antiforgery_FormToken_MustBeProvided_Generic" xml:space="preserve">
-    <value>The form token must be provided.</value>
+  <data name="Antiforgery_HeaderToken_MustBeProvided" xml:space="preserve">
+    <value>The required antiforgery header value "{0}" is not present.</value>
+  </data>
+  <data name="Antiforgery_RequestToken_MustBeProvided" xml:space="preserve">
+    <value>The required antiforgery form field "{0}" and header value "{1}" are not present.</value>
```

Comment:
Is this correct? ..thought should have said something like `The required antiforgery token was not present either in form field "{0}" or header value "{1}"`?